src/components/ChatComponents/SuggestedPrompts.tsx

```diff
@@ -0,0 +1,139 @@
+import { ChainType } from "@/chainFactory";
+import { VAULT_VECTOR_STORE_STRATEGY } from "@/constants";
+import React, { useMemo } from "react";
+
+interface NotePrompt {
+  title: string;
+  prompts: string[];
+}
+const SUGGESTED_PROMPTS: Record<string, NotePrompt> = {
+  activeNote: {
+    title: "Active Note Insights",
+    prompts: [
+      `Provide three follow-up questions worded as if I'm asking you based on {activeNote}?`,
+      `What key questions does {activeNote} answer?`,
+      `Give me a quick recap of {{activeNote}} in two sentences.`,
+    ],
+  },
+  quoteNote: {
+    title: "Note Link Chat",
+    prompts: [
+      `Based on [[noteName]], what improvements should we focus on next?`,
+      `Summarize the key points from [[noteName]]`,
+      `Summarize the recent updates from [[noteName]].`,
+    ],
+  },
+  fun: {
+    title: "Test LLM",
+    prompts: [
+      `9.11 and 9.8, which is bigger?`,
+      `What's the longest river in the world?`,
+      `If a lead ball and a feather are dropped simultaneously from the same height, which will reach the ground first?`,
+    ],
+  },
+  qaVault: {
+    title: "Vault Q&A",
+    prompts: [
+      `What are the main takeaways from my recent project notes?`,
```

Comment:
These 3 questions will likely fail in Vault QA because "recent project notes", "deadline", "brainstorming sessions" may not exist literally in the user's vault. They are more appropriate as Copilot Plus questions.

Reply:
I'm struggling to come up with meaningful prompts that's more generic than this. I changed them and ask user to fill specific topic or concept with `<topic>` and `<concept>` placeholder. Similar to what you pointed out in another comment, we can train user to understand `<string>` as an indicator that this needs to be replaced. We can also swap the out when the prompt is inserted into the input and place the cursor in place. In the scope of this PR, I suggest just trying the new prompt for now. PTAL